src/cards/light-card/controls/light-color-temp-control.ts

```diff
@@ -25,10 +27,16 @@ export class LightColorTempControl extends LitElement {
 
         const colorTemp = getColorTemp(this.entity);
 
+        const sliderStyle = {};
+        if (state === "off") {
+            sliderStyle["--gradient"] = "none";
+        }
+
         return html`
             <mushroom-slider
+                style=${styleMap(sliderStyle)}
                 .value=${colorTemp}
-                .disabled=${state !== "on"}
+                .disabled=${!isAvailable(this.entity)}
```

Comment:
What about adding a new property `inactive` and replacing the `disabled` class by `inactive` class in the slider ? 
The will be 2 properties for 2 different purpose :
- `disabled` : disable the interaction with the slider
- `inactive` : change style for inactive style for the slider

What do you think about that?


It can avoid to pass sliderStyle to the slider.

```suggestion
                .disabled=${!isAvailable(this.entity)}
                .inactive=${!isActive(this.entity)}
```

Reply:
Seems good to me.  If can't carve out some time today I should be able to make those changes tonight.

Reply:
Yep we can rename it to `--bg-color-inactive` and `--main-color-inactive` for consistency 🙂